mainwindow.h

```diff
@@ -26,16 +25,15 @@ private slots:
 
 private:
     Ui::MainWindow *ui;
-    Choices player1Choice;
-    Choices player2Choice;
     int roundCount = 1;
     int winCount = 0;
```

Comment:
I believe these values are belong to player class :))

Reply:
I have moved them into player class.

Reply:
Done